### PR TITLE
page object for Integration_page

### PR DESCRIPTION
**Pull request by qaivanpleten**
Changed test for Integration page using page object methodology. Created integration_page file (page object) and changed test_integration_page.py
#18

### CODE REVIEW

File: sp_at/pages/integration_page.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        self._url.open()
+
+    def header(self) -> Header:
+        return self._header
+
+    def body(self) -> Body:
+        return self._body
+
+
+class SlowIntegrationPage(IntegrationPage):
```
Comment (extsoft) [inline sp_at/pages/integration_page.py]: You don't need this class if you don't use it.

File: sp_at/pages/integration_page.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    def __init__(self, driver: WebDriver):
+        self._title = ElementByXpath(driver, '//*[@id="content_left"]/h1')
+        self._what_next_block = ElementById(driver, 'content_right')
+        self._schedule_block = ElementById(driver, 'content_left')
+        self._schedule_button = ElementByXpath(driver, '//*[@id="content_right"]/a/button')
+
+    def present(self)-> bool:
+        for element in (self._title, self._what_next_block, self._schedule_block, self._schedule_button):
+            if element.instance().is_displayed():
+                return True
```
Comment (extsoft) [inline sp_at/pages/integration_page.py]: This line says that body is present if the title is displayed. I'm not sure that this is correct. I suppose you need to return `true` only if all elements are displayed.

File: sp_at/test_cases/test_integration_page.py [hunk 3]
```diff
@@ -4,41 +4,21 @@
 from sp_at.actions.general_actions import GeneralActions
 from sp_at.elements.integration_page_elements import IntegrationPageElements
 from sp_at.elements.main_page_elements import MainPageElements
```
Comment (extsoft) [inline sp_at/test_cases/test_integration_page.py]: spare import

File: sp_at/test_cases/test_integration_page.py [hunk 4]
```diff
@@ -4,41 +4,21 @@
 from sp_at.actions.general_actions import GeneralActions
```
Comment (extsoft) [inline sp_at/test_cases/test_integration_page.py]: spare import

**Review by extsoft — CHANGES_REQUESTED** [review]
@qaivanpleten please take a look at the comments below.

**Comment by qaivanpleten** [comment]
@extsoft 
The issues are fixed. Please check

**Review by extsoft — APPROVED** [review]

**Pull request merged**